azavea/echo-locator
# Staging site frontend changes

## Overview

Staging site frontend changes based off of feedback from user testing. Includes stylistic changes and some simple restructuring of components.

### Demo

Optional. Screenshots, `curl` examples, etc.


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output


Resolves #XXX
